Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -102,13 +99,6 @@ public float rollMoveSpeed
 	bool _isOnGround = false;
```

Comment:
I think this is something that's better suited for just enabling the debug inspector.